src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -0,0 +1,135 @@
+    <Compile Include="System\Net\Http\Headers\CacheControlHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\CacheControlHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\ContentDispositionHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\ContentRangeHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\DateHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\EntityTagHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\GenericHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\HeaderUtilities.cs" />
+    <Compile Include="System\Net\Http\Headers\HttpContentHeaders.cs" />
+    <Compile Include="System\Net\Http\Headers\HttpGeneralHeaders.cs" />
+    <Compile Include="System\Net\Http\Headers\HttpHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\HttpHeaders.cs" />
+    <Compile Include="System\Net\Http\Headers\HttpHeaderValueCollection.cs" />
+    <Compile Include="System\Net\Http\Headers\HttpRequestHeaders.cs" />
+    <Compile Include="System\Net\Http\Headers\HttpResponseHeaders.cs" />
+    <Compile Include="System\Net\Http\Headers\Int32NumberHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\Int64NumberHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\MediaTypeHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\MediaTypeHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\MediaTypeWithQualityHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\NameValueHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\NameValueWithParametersHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\ObjectCollection.cs" />
+    <Compile Include="System\Net\Http\Headers\ProductHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\ProductInfoHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\ProductInfoHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\RangeConditionHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\RangeHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\RangeItemHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\RetryConditionHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\StringWithQualityHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\TimeSpanHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\TransferCodingHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\TransferCodingHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\TransferCodingWithQualityHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\UriHeaderParser.cs" />
+    <Compile Include="System\Net\Http\Headers\ViaHeaderValue.cs" />
+    <Compile Include="System\Net\Http\Headers\WarningHeaderValue.cs" />
+
+    <!-- TODO: Must be moved to the System.Net.Common folder -->
```

Comment:
@davidsh Please update the comment: the files should eventually be moved to `\Common\System\Net`.